packages/blade/src/components/Box/BaseBox/types/responsive.types.ts

```diff
@@ -0,0 +1,11 @@
+import type { Breakpoints } from '~tokens/global';
+
+type MakeValueResponsive<T> =
```

Comment:
What does `MakeObjectResponsive` & `MakeValueResponsive` do? Can we add some comments here?

Reply:
added